LocalSettings.php

```diff
@@ -1007,12 +1007,10 @@
 		'default' => true,
 	],
 	'wgCreateWikiUseSecureContainers' => [
-		'default' => false,
```

Comment:
Why is this true by default? it'll mean all wikis will be created as private containers if setContainersAccess.php is run.

Reply:
> Why is this true by default? it'll mean all wikis will be created as private containers if setContainersAccess.php is run.

If true, it enables them on private wikis.  (except deleted and temp which is always private). Not all wikis.